multi-agent/tests/unit/blueprints/test_blueprint_service_versioning.py

```
@@ -0,0 +1,417 @@
+        svc = _make_service(repo=repo, version_repo=version_repo)
+        return svc, repo, version_repo
+
+    def test_returns_true_on_success(self):
+        svc, _, _ = self._setup()
+        result = svc.restore_version("bp-1", target_version=2, user_id="bob")
+        assert result is True
+
+    def test_restores_spec_from_historic_snapshot(self):
+        """The restore must pass the historic snapshot's spec_dict to update_draft()."""
+        historic_spec = {"name": "Restored Name", "nodes": [{"id": "n1"}]}
+        svc, repo, version_repo = self._setup(target_spec=historic_spec)
```

Ruff (0.15.17)

[warning] Unpacked variable `version_repo` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)